src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -545,20 +547,12 @@ internal void OnNewCellClicked()
         editor.CreateNewMicrobe();
     }
 
-    internal void ToggleHelpScreen()
+    internal void OpenHelpScreen()
```

Comment:
I think this and that other toggle method renaming is not the best, as this method actually toggles the help screen, but always hides the menu...